src/components/CardEditor/CardEditForm/_card-edit-form.scss

```diff
@@ -157,13 +157,31 @@
     margin-bottom: 0;
   }
 
-  &-input {
+  &--input {
     margin-bottom: $spacing-05;
+    position: relative;
   }
 
   &-btn {
     width: 100%;
   }
+  &-clear-button {
+    position: absolute;
+    right: $spacing-05;
+    margin-bottom: 0;
+    padding: 0 $spacing-02;
+    bottom: $spacing-04;
+    color: $ui-05;
```

Comment:
I think this positioning might be redundant if we used the `size="field"` prop on the button. I believe it's intended to size buttons for use in fields like this. Nothing to hold this up for, but would be nice to in the future have it "full bleed" without any padding like other buttons. pagination for example

![image](https://user-images.githubusercontent.com/3360588/101089827-56e48b00-357b-11eb-9e34-bfb44b7e181f.png)